### PR TITLE
adding version file, similar to foreman-installer

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+2.4.0-develop
```
Comment (ehelms) [inline VERSION]: develop?

**Comment by ehelms** [comment]
I am assuming we want to:

1) Move packaging to katello-packaging
2) Then make this change https://github.com/Katello/katello-installer/pull/245

**Comment by stbenjam** [comment]
>  I am assuming we want to:
> 1) Move packaging to katello-packaging
> 2) Then make this change #245

:+1:

**Comment by ehelms** [comment]
@jlsherrill do we still need this or will #245 handle it?

**Comment by stbenjam** [comment]
I have to update the Rakefile to read VERSION anyway, so I included it in #245.

**Comment by stbenjam** [comment]
We also need to blow away katello-installer.spec and rel-eng, when you're ready.

**Comment by ehelms** [comment]
@stbenjam done -- https://github.com/Katello/katello-installer/pull/250

**Comment by ehelms** [comment]
I think this can be closed now.

**Comment by ehelms** [comment]
Closing as this is done already

**Pull request closed without merging**